### DISCUSSION TITLE
Improve members site

### DISCUSSION THREAD

**Discussion by vinayak-trivedi** [Real-Dev-Squad/website-members#439] [General]
There are three things on our members site that I think should get done:

- We should `refactor` our code
- We should have `tests` for everything.
- improve the performance of the site overall

Everyone, please provide suggestions below, on what we can improve. what are the problems that you think should be corrected?

**Comment by singh3069** [comment]
Agree with you, mostly with the `refactor `our code part

**Comment by yesyash** [comment]
Move from `JavaScript` to `Typescript`.
